app/helpers/event_helper.rb

```diff
@@ -0,0 +1,17 @@
+module EventHelper
+  def self.select_color(index=0)
```

Comment:
Methods in `app/helpers/*_helper.rb` modules tend to be instance methods.

These Helper instance methods can be accessed from within view templates without the module name which tends to give a little more concise view templates.

```diff
-  def self.select_color(index=0)
+  def select_color(index=0)
```

```diff
- <tr style="background-color:<%= EventHelper.select_color(i) %>" >
+ <tr style="background-color:<%= select_color(i) %>" >
```